examples/benchmarks/memory_bw_latency_performance.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Micro benchmark example for disk performance.
+
+Commands to run:
+  python3 examples/benchmarks/memory_bw_latency_performance.py
+"""
+
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.common.utils import logger
+
+if __name__ == '__main__':
+    context = BenchmarkRegistry.create_benchmark_context(
+        'memory-bw-latency-benchmark', platform=Platform.CPU, parameters='--tests bandwidth_matrix latency_matrix max_bandwidth'
```

Comment:
Make sure `mlc` is somewhere accessible like `SB_MICRO_PATH/bin`

Reply:
Yes, that's great